### PR TITLE
Filled in the blanks to break car interior down into sub-objects.

### PR DESCRIPTION
**Pull request by remingtonchan**

### CODE REVIEW

**Review by matkatmusic — APPROVED** [review]

**Review by matkatmusic — APPROVED** [review]

File: main.cpp [hunk 1]
```diff
@@ -12,76 +12,202 @@ What you create in this project will be used as the basis of Project 5 in the co
...
+5 properties:
+    1) types of broths
+    2) broth volume storage
+    3) broth preparation duration
+    4) number of staff
+    5) staff cycle duration
+3 things it can do:
+    1) create different types of broth
+    2) store broth
+    3) accept custom orders
```
Comment (matkatmusic) [inline main.cpp:152]: you probably mean `process custom order`. 
`accept custom orders` is a "Yes it does accept them/No it does not accept them", which can be conveyed with a boolean variable.
- [ ] revise

File: main.cpp [hunk 2]
```diff
@@ -12,76 +12,202 @@ What you create in this project will be used as the basis of Project 5 in the co
...
+/*
+Thing 8) Cooking Station
+5 properties:
+    1) number of cooks
+    2) number of stoves
+    3) number of pots
+    4) max number of bowls prepared at a time
+    5) seating capacity
+3 things it can do:
+    1) seat customers
```
Comment (matkatmusic) [inline main.cpp:178]: the cooking station seats customers? 
What kind of restaurant is this where the cooks interact with customers directly?
- [ ] revise

File: main.cpp [hunk 3]
```diff
@@ -12,76 +12,202 @@ What you create in this project will be used as the basis of Project 5 in the co
...
+Thing 9) Delivery Station
+5 properties:
+    1) number of motorcycles
+    2) number of drivers
+    3) max distance covered
+    4) max truck parking space capacity
+    5) number of receiving staff
+3 things it can do:
+    1) deliver ramen to customers
+    2) accept schedules deliveries
```
Comment (matkatmusic) [inline main.cpp:193]: again, `accept ___` can be conveyed with a boolean, which means its not an action.  
perhaps you just meant `schedules delivery`? 
- [ ] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 2 above)
Comment (remingtonchan) [reply main.cpp:178]: Hibachi? 😅 but got it, let me figure something out

File: main.cpp (hunk 1 above)
Comment (remingtonchan) [reply main.cpp:152]: I think I wanted to say `receive` here, but `process` is a way better term. Thanks!

**Review by matkatmusic — APPROVED** [review]

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**